src/main/scala/com/gu/ophan/backfill/ManifestFileStep.scala

```diff
@@ -0,0 +1,60 @@
+package com.gu.ophan.backfill
+
+import java.time.LocalDate
+
+import upickle.default._
+import java.nio.channels.Channels
+import com.google.cloud.storage.{ BlobId, BlobInfo, StorageOptions }
+
+object ManifestFileStep extends JsonHandler[Seq[JobConfig], String] {
+
+  override def process(input: Seq[JobConfig])(implicit env: Env): String = {
+    (for (cfg <- input.headOption) yield {
+      logger.info("creating manifest file")
+
+      // if we use the same function we will definitely end up in the same place
+      val prefix = ExtractDataStep.pathPrefix(cfg)
+
+      val manifestLines = input.map { jobConfig =>
+        ManifestFileLine(jobConfig.startDateInc, jobConfig.documentCount.get)
+      }
+
+      uploadObject(prefix = prefix, manifest = manifestLines)
+    }).getOrElse("<<EMPTY>>")
+  }
+
+  def uploadObject(
+    prefix: String,
+    manifest: Seq[ManifestFileLine],
+    projectId: String = "datatech-platform-prod",
+    bucketName: String = "gu-ophan-backfill-prod",
```

Comment:
Could we construct this bucket name in a common val/function? The bucket name's also used in this bit of code:

https://github.com/guardian/ophan-backfill-step-function/blob/72cbbe453c24e8be58b9803b6b556e4e408d4002/src/main/scala/com/gu/ophan/backfill/ExtractData.scala#L19

...I can see we're already handling the prefix with `ExtractDataStep.pathPrefix(cfg)`!